package.json

```diff
@@ -72,6 +72,7 @@
     "gulp-bump": "2.7.0",
     "gulp-conventional-changelog": "1.1.3",
     "gulp-eslint": "3.0.1",
+    "i": "0.3.6",
```

Comment:
What is this for?

Reply:
it seems to have been updated when I run npm install